graylog2-web-interface/webpack.config.js

```diff
@@ -156,8 +148,8 @@ if (TARGET === 'test') {
   console.log('Running test/ci mode');
   module.exports = merge(webpackConfig, {
     module: {
-      preLoaders: [
-        { test: /\.js(x)?$/, loader: 'eslint-loader', exclude: /node_modules|public\/javascripts/ }
+      rules: [
+        { test: /\.js(x)?$/, enforce: "pre", loader: 'eslint-loader', exclude: /node_modules|public\/javascripts/ }
```

Comment:
We also have some in the `start` task output, we should replace them as well for consistency.